File: pkg/models/token.go
package models

import "time"

type Token struct {
	ID                     string
	UserID                 string
	AccessToken            string
	RefreshToken           string
	AccessTokenType        string
	RefreshTokenExpiresAt  time.Time
	RefreshTokenLastUsedAt time.Time
	AccessTokenExpiresAt   time.Time
	IssuedAt               time.Time
	Scope                  string
	GrantType              string
	RevokedAt              time.Time
}

File: admin-ui/src/types/token.ts
export interface AdminTokenResponse {
  id: string;
  user_id: string | null;
  username: string;
  email: string;
  scope: string;
  grant_type: string;
  access_token_expires_at: string;
  issued_at: string;
  revoked_at: string | null;
  status: string;
}
